update/Pos/Update/Poll/RollTrans.hs

```diff
@@ -74,8 +74,10 @@ instance MonadPoll m => MonadPoll (RollT m) where
         insertIfNotExist id unChangedPropsL getProposal
         deactivateProposal id
 
-    setSlottingData =
-        setValueWrapper unSlottingDataL getSlottingData setSlottingData
+    putEpochSlottingData ei esd = ether $ do
+        insertIfNotExist ei unSlottingDataL getEpochSlottingData
+        putEpochSlottingData ei esd
+
```

Comment:
You didn't provide an explicit definition for `delEpochSlottingData`. It's correct (it will be `delEpochSlottingData = lift . delEpochSlottingData` by default), but can you write it explicitly as it's done for `delConfirmedSV`?
```
    -- can't be called during apply
    delConfirmedSV = lift . delConfirmedSV
```